.github/workflows/mlc_config.json

```diff
@@ -0,0 +1,6 @@
+{
+  "retryOn429": true,
+  "retryCount": 5,
+  "fallbackRetryDelay": "30s",
+  "aliveStatusCodes": [200, 403]
+} 
```

Comment:
Additional configuration for the first link checker (s. https://github.com/gaurav-nelson/github-action-markdown-link-check for details)
It seems that some sites (e.g. github itself) block such script actions. In such a case, if the link is valid but the action cannot access it, 403 code is returned. So I added it to the list of allowed return codes (200 is: link is valid)